src/objects.c

```diff
@@ -494,6 +543,17 @@ PgUser *find_user(const char *name)
 	return user;
 }
 
+/* find priority by name */
+uint16_t find_priority_for_application(const char *app_name)
```

Comment:
App names tend to be _super ugly_, which is going to make exact matches hard. Example application names from our prod environment are: 
```
 resque-1.27.4: Processing proc...499450757 [QueuedProcoreEmail]
 Passenger RubyApp: /data/procore/current (production)
 resque-1.27.4: Processing proc...499449561 [QueuedProcoreEmail]
 resque-1.27.4: Processing proc...499450363 [QueuedProcoreEmail]
 psql
 sidekiq 4.1.2 blue [5 of 5 busy]
 resque-1.27.4: Processing proc...499450545 [QueuedProcoreEmail]
 resque-1.27.4: Processing proc...499452283 [QueuedProcoreEmail]
 resque-1.27.4: Processing proc...499449819 [QueuedProcoreEmail]
 sidekiq 4.1.2 blue [4 of 5 busy]
 resque-1.27.4: Processing proc...499450301 [QueuedProcoreEmail]
 sidekiq 4.1.2 green [0 of 5 busy]
```

Do we have any reasonable to implement options for non-exact matches?